Resources/Prototypes/Entities/Structures/Furniture/beds.yml

```diff
@@ -39,7 +43,37 @@
         spawn:
           SheetSteel1:
             min: 1
-            max: 1
-          MaterialCloth1:
+            max: 2
+
+- type: entity
+  parent: Bed
+  id: DogBed
+  name: dog bed
+  description: A comfy-looking dog bed. You can even strap your pet in, in case the gravity turns off.
+  components:
+  - type: Sprite
+    state: dogbed
+  - type: Strap
+    position: Stand
+    rotation: 0
+  - type: Construction
```

Comment:
Should dog beds really be buckleable?

Reply:
yes

Reply:
Try have to be otherwise you can't buckle pets to them which is their practices point. Being able to strap people to them is a bonus I suppose?